src/style.css

```
@@ -1,3 +1,92 @@
+:root {
+  --basic-fonts-color: #000;
+  --background-color: #fff;
+  --main-active-color: #00acdc;
+  --font-size: 12px;
+  --line-height: 14px;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: var(--font-size);
+  color: var(--basic-fonts-color);
+  background-color: var(--background-color);
+}
+
 body {
   margin: 0;
 }
+
+ul,
+li {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a,
+a:visited {
+  text-decoration: none;
+}
+
+.header__wrapper {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.header__list {
+  display: flex;
+}
+
+.header__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  transition: color 0.3s ease;
+  height: 60px;
+  font-weight: 500;
+  font-size: var(--font-size);
+  line-height: 60px;
+  text-transform: uppercase;
+  text-align: center;
+  align-items: center;
+  color: var(--basic-fonts-colorn);
```

```suggestion
  display: flex;
  align-items: center;
  
  height: 60px;
  
  font-size: var(--font-size);
  font-weight: 500;
  line-height: 60px;
  text-align: center;
  text-transform: uppercase;
  color: var(--basic-fonts-color);
  
  transition: color 0.3s ease;
```

The order of your styles is really important for better readability